Documentation/ApiOverview/ContentSecurityPolicy/_csp_example_com.yaml

```
@@ -0,0 +1,23 @@
+# Inherits default site-unspecific frontend policy mutations (enabled per default)
+inheritDefault: true
+mutations:
+  # Allow iframes to anywhere (HTTPS only, not HTTP)
```

```suggestion
  # Allow frames/iframes to anywhere (HTTPS only, not HTTP)
```

This makes it more clear.